logging/fluentd-sidecar-es/config_generator.sh

```diff
@@ -21,7 +21,7 @@ fi
 
 for filepath in $FILES_TO_COLLECT
 do
-  filename=$(basename $filepath)
+  filename=`echo $filepath|sed -e "s/\//_/g"`
```

Comment:
This could still lead to collisions if files have underscore in the name. Unlikely, but possible.

Can you check if the target file exists & add perhaps an incrementing suffix if it does?